cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/DynamicDatasetContext.java

```diff
@@ -72,16 +76,28 @@
    * arguments.
    *
    * @param name the name of the dataset
+   *
    * @return runtime arguments for the given dataset
    */
   protected Map<String, String> getRuntimeArguments(String name) {
     return RuntimeArguments.extractScope(Scope.DATASET, name, runtimeArguments);
   }
 
+  /**
+   * @param namespace the {@link Id.Namespace} in which the transaction context is created
```

Comment:
I simply copied this from the other constructor, because this one was missing the javadocs. Not sure what is meant by this. Perhaps it should say, "in which the dataset context is created"